blog/views.py

```diff
@@ -1,8 +1,19 @@
+"""Views for blog app.
+
+- post_list - returns post list;
+"""
+
 from django.shortcuts import render
 from .models import Post
 from django.utils import timezone
 # Create your views here.
 
+
 def post_list(request):
-    posts = Post.objects.filter(published_date__lte=timezone.now()).order_by('published_date')
-    return render(request, 'blog/post_list.html', {'posts' : posts})
+    """Show posts, which are published, ordered by published date."""
+    posts = (
+        Post.objects
```

Comment:
##### Just a tip:
Sometimes I leave the dot in the end of previous line when wrapping expression with parentheses.
Also you could leave this line right after the left parenthesis, adjust all following lines with its position and keep the right parenthesis in the end of the last line of this expression.
While normally it's up to you to decide how to indent your code (unless you violate PEP 8), usually you'll follow the style guide, which the whole team decided to use.